static/locales/en-US.yaml

```diff
@@ -18,69 +18,69 @@ Delete: Delete
 Select all: Select all
 Reload: Reload
 Force Reload: Force Reload
-Toggle Developer Tools: Toggle Developer Tools
+Toggle Developer Tools: Developer Tools
 Actual size: Actual size
 Zoom in: Zoom in
 Zoom out: Zoom out
-Toggle fullscreen: Toggle fullscreen
+Toggle fullscreen: Fullscreen
 Window: Window
 Minimize: Minimize
 Close: Close
 Back: Back
 Forward: Forward
 Open New Window: Open New Window
 
-Version $ is now available!  Click for more details: Version $ is now available!  Click
-  for more details
+Version $ is now available!  Click for more details: Version $ is now available! Click
+  for details.
 Download From Site: Download From Site
-A new blog is now available, $. Click to view more: A new blog is now available, $.
-  Click to view more
+A new blog is now available, $. Click to view more: New blogpost available, $.
```

Comment:
I looked up and didn't find example of blogpost as a word, should be "blog post".